src/hooks/restrict-to-owner.js

```diff
@@ -0,0 +1,42 @@
+import errors from 'feathers-errors';
+
+const defaults = {
+  idField: '_id',
+  ownerField: 'userId'
+};
+
+export default function(options = {}){
+  return function(hook) {
+    if (hook.type !== 'before') {
```

Comment:
You thinking that it removes any documents that you don't own? That's pretty easy to do.